SuperBuild.cmake

```
@@ -338,7 +338,8 @@ list_conditional_append(Slicer_BUILD_LandmarkRegistration Slicer_REMOTE_DEPENDEN
 
 Slicer_Remote_Add(SurfaceToolbox
   GIT_REPOSITORY "${EP_GIT_PROTOCOL}://github.com/Slicer/SlicerSurfaceToolbox"
-  GIT_TAG 5fde114b1f950f632e13a658c44ca948e7955e89
+  # GIT_TAG 5fde114b1f950f632e13a658c44ca948e7955e89
+  GIT_TAG 6b9fe7c99d9b4c78ae6992b2f44caed8ec3c5059 # Needed for AUTOMOC support until #79 of SlicerSurfaceToolbox is merged
```

Per https://github.com/Slicer/SlicerSurfaceToolbox/commit/66669791bb88b5e21592aa3eeecddbe334fe9080
```suggestion
  GIT_TAG 66669791bb88b5e21592aa3eeecddbe334fe9080
```

---

Done!